svn/common.py

```diff
@@ -28,7 +29,7 @@ def __init__(self, url_or_path, type_, *args, **kwargs):
         self.__trust_cert = kwargs.pop('trust_cert', None)
 
         if type_ not in (svn.constants.LT_URL, svn.constants.LT_PATH):
-            raise ValueError("Type is invalid: %s" % (type_))
+            raise SvnException("Type is invalid: %s" % type_)
```

Comment:
`SvnException` as the parent class of all exceptions sounds like a good idea to me. I would still strongly suggest to differentiate between the exception raised when the user passes an incorrect parameter, and when the CLI client returns an error code.